libevm/EVMC.cpp

```diff
@@ -21,6 +21,36 @@ EVM::EVM(evmc_instance* _instance) noexcept : m_instance(_instance)
         m_instance->set_option(m_instance, pair.first.c_str(), pair.second.c_str());
 }
 
+EVMC::EVMC(evmc_instance* _instance) : EVM(_instance)
+{
+    static constexpr auto tracer = [](evmc_tracer_context * context, int depth, int step,
+        size_t code_offset, evmc_status_code status_code, int64_t gas_left, size_t stack_num_items,
+        const evmc_uint256be* pushed_stack_item, size_t memory_size,
+        size_t changed_memory_offset, size_t changed_memory_size,
+        const uint8_t* changed_memory) noexcept {
+
+        EVMC* evmc = reinterpret_cast<EVMC*>(context);
+
+        // TODO: It might be easier to just pass instruction from VM.
+        InstructionInfo instrInfo = instructionInfo(static_cast<Instruction>(evmc->m_code[code_offset]));
+
+        std::cerr << "EVMC " << depth << " " << step << " " << code_offset << " " << instrInfo.name << " " << status_code
+                  << " " << gas_left << " " << stack_num_items;
+
+        if (pushed_stack_item)
+            std::cerr << " +[" << fromEvmC(*pushed_stack_item) << "]";
+
+        std::cerr << " " << memory_size << "\n";
+
+        (void)changed_memory_offset;
```

Comment:
Prefer not naming unused params instead of this https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#f9-unused-parameters-should-be-unnamed